library/std/src/lib.rs

```diff
@@ -373,6 +373,7 @@
 #![feature(sync_unsafe_cell)]
 #![feature(temporary_niche_types)]
 #![feature(ub_checks)]
+#![feature(unsigned_signed_diff)]
```

Comment:
this change makes sense (it's a library feature) but it is orthogonal to the stabilization. could you make a separate PR for it?